composer.json

```diff
@@ -17,10 +17,14 @@
             "email": "leizmo@gmail.com"
         }
     ],
+    "require": {
+        "php": "^7.0"
+    },
     "require-dev": {
-        "phpunit/phpunit": "^6.5.0"
+        "phpunit/phpunit": "^6.5.0",
+        "friendsofphp/php-cs-fixer": "^2.15"
```

Comment:
Define `php-cs-fixer` dependency inside `require-dev`.

When setting up the package development environment, this will be installed.